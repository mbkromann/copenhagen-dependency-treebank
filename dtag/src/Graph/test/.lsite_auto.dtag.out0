undef
30
30
