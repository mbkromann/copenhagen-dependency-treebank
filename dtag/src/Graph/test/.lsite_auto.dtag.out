PostScript files loaded from /opt/dtag/
undef
undef
undef

WARNING! You have unsaved graphs!
Type 'exit' or 'exit!' if you really want to quit...

